sdk/cognitiveservices/azure-cognitiveservices-formrecognizer/DESIGN_DOC.md

```diff
@@ -0,0 +1,544 @@
+    page_number: int
+    angle: float
+    width: int
+    height: int
+    unit: str
+    language: str
+    lines: List[ExtractedLine]
+```
+
+### Custom Models Labeled
+```python
+# Training ---------------------------------------------------
+class CustomLabeledModel:
+    model_id: str
+    status: str
+    created_on: ~datetime.datetime
+    last_updated_on: ~datetime.datetime
+    train_result: LabeledTrainResult
+
+class LabeledTrainResult:
+    fields: List[FieldNames]
+    average_model_accuracy: float
+    documents: List[TrainingDocumentInfo]
+    errors: List[FormRecognizerError]
+
+class TrainingDocumentInfo:
+    document_name: str
+    status: str
+    page_count: int
+    errors: List[FormRecognizerError]
+
+class FieldNames:
+    field_name: str
+    accuracy: float
+
+class FormRecognizerError:
+    code: str
+    message: str
+
+# Analyze ---------------------------------------------------
+class ExtractedForm:
+    labels: Dict[str, LabelValue]
+    pages: List[ExtractedFormPage]
+    page_range: PageRange
+
+class ExtractedFormPage:
+    tables: List[ExtractedTable]
+    page_metadata: PageMetadata
+    page_number: int
+
+class LabelValue:
+    text: str
+    value: Union[int, float, str, datetime]
+    bounding_box: List[float]
+    confidence: float
+    page_number: int
+    elements: List[Union[ExtractedLine, ExtractedWord]]
+
+class ExtractedLine:
+    text: str
+    bounding_box: List[float]
+    language: str
+    page_number: int
+    words: List[ExtractedWord]
+
+class ExtractedWord:
+    text: str
+    bounding_box: List[float]
+    confidence: float
+    page_number: int
+    line: ExtractedLine
+
+class PageMetadata:
+    page_number: int
+    angle: float
+    width: int
+    height: int
+    unit: str
+    language: str
+    lines: List[ExtractedLine]
+```
+
+
+### List/Get/Delete Models
+```python
+class ModelInfo:
+    model_id: str
+    status: str
+    created_on: ~datetime.datetime
+    last_updated_on: ~datetime.datetime
+
+class ModelsSummary:
+    count: int
+    limit: int
+    last_updated_on: ~datetime.datetime
+```
+
+
+### Custom Training Samples
+
+#### Custom: Train and Analyze without labels
+```python
+from azure.ai.formrecognizer import CustomFormClient
+
+client = CustomFormClient(endpoint=endpoint, credential=credential)
+
+# Train
+blob_sas_url = "xxxxx"  # training documents uploaded to blob storage
+poller = client.begin_training(blob_sas_url)
+model = poller.result()
+
+# Custom model information
+print("Model ID: {}".format(model.model_id))
+print("Status: {}".format(model.status))
+print("Created on: {}".format(model.created_on))
+print("Last updated on: {}".format(model.last_updated_on))
+
+# Training result information
+train_result = model.train_result
+for document in train_result.documents:
+    print(document.name)
+    print(document.status)
+    print(document.page_count)
+    print(document.errors)
+
+print("Form type ID: {}".format(train_result.form_type_id))
```

Comment:
`form_type_id` should be part of the `extracted_fields[i]` info.

Reply:
Correct, fixed this in sample